code/datums/supply_packs/operations.dm

```diff
@@ -1,117 +1,151 @@
-//the idea is to put all the bulk items scanner secure crate with lot's of flares MRE in it and at the end OB and non buyable.
+//large item crates
 
-//non buyable
+/datum/supply_packs/floodlight
```

Comment:
No floodlights, this just leads to issues currently.

Reply:
removed